### PR TITLE
import deprecate from ember-deprecations

### PR DESCRIPTION
**Pull request by BryanCrotaz**

### CODE REVIEW

**Comment by igorT** [comment]
Do we need to add a dependency on ember-deprecations?

**Comment by BryanCrotaz** [comment]
it used to be imported from @ember/debug, but that doesn't work any more.

**Comment by igorT** [comment]
@BryanCrotaz my question is more, how come we import from `ember-deprecations` but don't add it to `package.json`?

**Comment by snewcomer** [comment]
Also, do we have a link or RFC describing this change in behaviour?  I'm also unable to find this package.

It looks like this function is still around...

https://api.emberjs.com/ember/3.24/functions/@ember%2Fdebug/deprecate

File: packages/-ember-data/addon/setup-container.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
-import { deprecate } from '@ember/debug';
```
Comment (rwjblue) [inline packages/-ember-data/addon/setup-container.js:1]: The original import path is correct, we should not change it.

Can you explain what made you want to update?

**Review by rwjblue — CHANGES_REQUESTED** [review]

**Comment by runspired** [comment]
Concurring with @rwjblue import from `@ember/deprecate` is the correct location. https://api.emberjs.com/ember/3.25/functions/@ember%2Fdebug/deprecate

Some more context on the issue this was attempting to address in a bug ticket would be helpful if this has remained an issue for your app.

**Pull request closed without merging**